### PR TITLE
Use env variables

### PR DESCRIPTION
**Pull request by agebhardt**
Creating a config object that will allow us to set various environment variables. Will look at variables passed in via the command line (on script execution) first then fall back to a local `.env` file (<-- this will come in handy with local dev environments). At the moment, this just looks for a user, but we can expand upon this. If it can't find a user, it fails with an error.

### CODE REVIEW

**Review by cat5inthecradle — APPROVED** [review]
I think it should be called `REDSHIFT_CLUSTER_URL` or something like that, but that's irrelevant for this PR.

File: config.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    # fall back to .env file
+    if(user is None):
+      load_dotenv(find_dotenv())
+      user = os.environ.get("REDSHIFT_USER")
+
+    # couldn't find a REDSHIFT_USER, fail
+    if(user is None):
+      raise Exception("REDSHIFT_USER not found")
+
+    self.REDSHIFT_USER = user
```
Comment (cat5inthecradle) [inline config.py:22]: would we need to repeat this for each variable?

Comment (agebhardt) [reply config.py:22]: In theory 😅 I'm sure there's a way to clean it up, just didn't do that for this PR.

Comment (agebhardt) [reply config.py:22]: I suspect we can make a method on this class that has a generic "get given variable" and we just call that for each one. I can work on that next 😄

**Pull request merged**